### ISSUE TITLE
WS of Deformable Convolution.

### ISSUE THREAD

**Issue by lizhen2017** [joe-siyuan-qiao/WeightStandardization#10]
Have you done any experiments on DFC? I guess that both offset convolution and deformable convolution can use WS. Looking forward to your reply.

**Comment by joe-siyuan-qiao** [comment]
No, we haven't. mmdetection (https://github.com/open-mmlab/mmdetection) includes both ws and deformable convolution that can be easily used together.

**Comment by lizhen2017** [comment]
I've seen the mmdetection source code, only find the ws_conv, the conv_type parameter works only for conv_2d, maybe be I didn't find that.

**Comment by joe-siyuan-qiao** [comment]
You can follow these steps to use them together:

1. Comment https://github.com/open-mmlab/mmdetection/blob/d2483e15fc48b4166815c15a6e12be864bcc521a/mmdet/models/backbones/resnet.py#L163
2. Add ws to https://github.com/open-mmlab/mmdetection/blob/d2483e15fc48b4166815c15a6e12be864bcc521a/mmdet/ops/dcn/deform_conv.py#L235
3. Add dcn settings in https://github.com/open-mmlab/mmdetection/blob/d2483e15fc48b4166815c15a6e12be864bcc521a/configs/dcn/faster_rcnn_dconv_c3-c5_r50_fpn_1x.py to https://github.com/open-mmlab/mmdetection/blob/d2483e15fc48b4166815c15a6e12be864bcc521a/configs/gn%2Bws/faster_rcnn_r50_fpn_gn_ws_1x.py.
